app/views/api/v2/hosts/main.json.rabl

```diff
@@ -9,7 +9,7 @@ attributes :ip, :environment_id, :environment_name, :last_report, :mac, :realm_i
            :enabled, :puppet_ca_proxy_id, :managed, :use_image, :image_file, :uuid, :compute_resource_id, :compute_resource_name,
            :compute_profile_id, :compute_profile_name, :capabilities, :provision_method,
            :puppet_proxy_id, :certname, :image_id, :image_name, :created_at, :updated_at,
-           :last_compile, :puppet_status
+           :last_compile, :global_status
```

Comment:
Can you keep both?  Alias puppet_status to global_status, and add global_status too.

You'll probably want to add some child nodes for the more detailed statuses.